blender/arm/material/make_transluc.py

```diff
@@ -18,7 +18,7 @@
 
 
 def make(context_id):
-	con_transluc = mat_state.data.add_context({ 'name': context_id, 'depth_write': False, 'compare_mode': 'less', 'cull_mode': 'clockwise', \
+	con_transluc = mat_state.data.add_context({ 'name': context_id, 'depth_write': False, 'compare_mode': 'less', 'cull_mode': 'none', \
```

Comment:
What does this change fix?

Reply:
Nothing, I was trying to set up the glass material and forgot that.